docs/guide.md

```diff
@@ -110,6 +110,27 @@ where `<API_KEY>` refers to the full generated API key.
 
 Please refer to [HttpRequest.META](https://docs.djangoproject.com/en/2.2/ref/request-response/#django.http.HttpRequest.META) for more information on headers in Django.
 
+#### Custom keyword
```

Comment:
Since this is a narrower step towards customizing header behavior, I think we should put this between the "Authorization header" and "Custom header" section. WDYT?

Also depending on https://github.com/florimondmanca/djangorestframework-api-key/pull/175/files#r702067843, looks like we'd need to rephrase this section a bit.

Reply:
I have moved the documentation to your suggested location and updated it to reflect the new method. @florimondmanca